src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -189,13 +190,14 @@ private AIState initiate()
 
     private boolean incrementBlock()
     {
-        return job.getSchematic().incrementBlock();//method returns false if there is no next block (schematic finished)
+        //method returns false if there is no next block (schematic finished)
+        return job.getSchematic().incrementBlock();
     }
 
     private void loadSchematic()
     {
         WorkOrderBuild workOrder = job.getWorkOrder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workOrder' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)